README.md

```diff
@@ -25,15 +25,31 @@ In this challenge use `Test Driven Development` to build a RESTful API using Nod
 Demonstrate your understanding of this week's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
 1. In Jest, what are the differences between `describe()` and `it()` globals, and what are good uses for them?
-1. What is the point of `Test Driven Development`? What do you think about this approach?
-1. Mention three types of automated tests.
+		- 'Describe' is for testing components, implying that there are sub
```

Comment:
Pretty much. The main thing about describe is that it lets you group your tests by a single aspect or thing that your are testing, like a particular route or function